addon/data/selector-ui.js

```diff
@@ -52,13 +52,13 @@ const ui = (function () { // eslint-disable-line no-unused-vars
         instructions.className = "pageshot-preview-instructions";
         instructions.textContent = "Drag or click on the page to select a region. Press ESC to cancel.";
         this.overlayEl.appendChild(instructions);
-        let button = document.createElement("button");
+        let button = document.createElement("div");
```

Comment:
This is less semantic but there was some weird rendering error with using a button here